concordium-base.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 2.0
 
--- This file has been generated from package.yaml by hpack version 0.35.0.
+-- This file has been generated from package.yaml by hpack version 0.35.1.
```

Comment:
Just out of curiosity, is this because of a new stack version we're using or?

Reply:
Yes. `stack` version 2.9.3 uses `hpack` 0.35.1.